tests/driver.c

```diff
@@ -16,6 +17,42 @@ struct timespec start_time, end_time;
 char driver_tag_msg[NUM_TRYTES_TAG];
 ta_send_mam_res_t* res;
 
+typedef status_t (*Proxy_apis)(const iota_client_service_t* const service, const char* const obj, char** json_result);
+typedef status_t (*Proxy_apis_without_args)(const iota_client_service_t* const service, char** json_result);
+
+static struct iri_proxys_apis_s {
+  Proxy_apis api;                            // function pointer of IRI Proxy API with args
+  Proxy_apis_without_args api_without_args;  // function pointer of IRI Proxy API without args
+  const char* name;
+  const char* json;  // args which are passed to IRI API
+} iri_prxoy_apis_g[] = {
+    {api_attach_to_tangle, NULL, "attach_to_tangle",
+     "{\"command\":\"attachToTangle\","
+     "\"trunkTransaction\":\"" TRYTES_81_1 "\","
+     "\"branchTransaction\":\"" TRYTES_81_2 "\","
+     "\"minWeightMagnitude\":" STR(MWM) ",\"trytes\":[\"" TRYTES_2673_1 "\",\"" TRYTES_2673_2 "\"]}"},
+    {api_check_consistency, NULL, "check_consistency",
+     "{\"command\":\"checkConsistency\","
+     "\"tails\":[\"" TRYTES_81_2 "\",\"" TRYTES_81_3 "\"]}"},
+    {api_get_balances, NULL, "get_balances",
+     "{\"command\":\"getBalances\","
+     "\"addresses\":[\"" TRYTES_81_2 "\",\"" TRYTES_81_3 "\"],"
+     "\"threshold\":" STR(THRESHOLD) "}"},
+    {api_get_inclusion_states, NULL, "get_inclusion_states",
+     "{\"command\":\"getInclusionStates\","
+     "\"transactions\":[\"" TRYTES_81_2 "\",\"" TRYTES_81_3 "\"],"
+     "\"tips\":[\"" TRYTES_81_2 "\",\"" TRYTES_81_3 "\"]}"},
+    {NULL, api_get_neighbors, "get_neighbors", NULL},
+    {NULL, api_get_node_info, "get_node_info", NULL},
+    {api_get_trytes, NULL, "get_trytes",
+     "{\"command\":\"getTrytes\","
+     "\"hashes\":[\"" TRYTES_81_2 "\",\"" TRYTES_81_3 "\"]}"},
+    {api_remove_neighbors, NULL, "remove_neighbors",
+     "{\"command\":\"removeNeighbors\","
```

Comment:
Since we temporarily don't test this proxy api, the related Test case can be removed